tools/depends/download-files.include

```diff
@@ -0,0 +1,33 @@
+#
+HASH_FOUND = yes
+ifneq ($(SHA512),)
+  HASH_TYPE = sha512
+  HASH_TOOL ?= $(SHA512SUM)
+else ifneq ($(SHA256),)
+  HASH_TYPE = sha256
+  HASH_TOOL ?= $(SHA256SUM)
+else
+  HASH_FOUND = no
+endif
+ifeq ($(NATIVE_OS), osx)
+  HASH_TOOL = $(SHASUM)
```

Comment:
Also either we will want to set this to $(SHASUM) -a 512, or on the configure set SHASUM to include the -a 512 flag. Otherwise in the case below if no hash, it will generate a sha1 hash